packages/lesswrong/lib/collections/userMostValuablePosts/schema.ts

```diff
@@ -4,6 +4,7 @@ import { userOwns } from '../../vulcan-users/permissions';
 
 const schema: SchemaType<DbUserMostValuablePost> = {
   userId: {
+    nullable: false,
```

Comment:
I think this collection is only used by the EA forum, so should be reviewed by them